tid_project/src/pages/Frame4/CommentItem.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const CommentItemWrapper = styled.li`
+  border-radius: 16px 16px 0px 0px;
+  position: relative;
+  display: flex;
+  min-height: 72px;
+  width: 100%;
+  flex-direction: column;
+  justify-content: center;
+  @media (max-width: 991px) {
+    max-width: 100%;
+  }
+`;
+
+const CommentContent = styled.div`
+  z-index: 0;
+  display: flex;
+  width: 100%;
+  align-items: center;
+  gap: 16px;
+  justify-content: flex-start;
+  flex: 1;
+  flex-wrap: wrap;
+  height: 100%;
+  padding: 8px 16px;
+  @media (max-width: 991px) {
+    max-width: 100%;
+  }
+`;
+
+const AvatarWrapper = styled.div`
+  align-self: stretch;
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  justify-content: center;
+  width: 40px;
+  margin: auto 0;
+`;
+
```

Comment:
Well done on creating custom styled-components and giving them intuitive naming! 👌